src/LINEBot/Response.php

```diff
@@ -29,17 +29,21 @@ class Response
     private $httpStatus;
     /** @var string */
     private $body;
+    /** @var string[] */
+    private $headers;
 
     /**
      * Response constructor.
      *
      * @param int $httpStatus HTTP status code of response.
      * @param string $body Request body.
+     * @param string[] $headers
      */
-    public function __construct($httpStatus, $body)
+    public function __construct($httpStatus, $body, $headers)
```

Comment:
The only thing I would do, to not make this a major version upgrade, is to make `$headers` optional, e.g. `$headers = []`. This would introduce new functionality in a backwards-compatible manner (if you follow semver).